### PR TITLE
Update workflows.py

### PR DESCRIPTION
**Pull request by lucasmiranda42**
Fixed potential bug introduced by Cellpose returning different sets of values when loading custom models through `cellpose.models.CellposeModel()`.

`masks, _, _, _ = model.eval([input_image], diameter=None, channels=[2, 1])` and similar lines of code fail when using a custom model. The error seems to come from the `_read_cellpose_model()` method in the same file. When a pretrained model is used, `model.eval()` returns 4 elements indeed (masks, flows, styles, diams). When a custom model is loaded using `models.CellposeModel()`, however, diameters are not returned (not sure why, tho, but their API [docs](https://cellpose.readthedocs.io/en/latest/api.html#cellpose.models.Cellpose.eval) may yield some insights). 

The introduced fix aims to take the first element of the returned tuple, which corresponds to the `masks` regardless of the class used to generate it. In my local tests this fixed the issue, but I can't guarantee this doesn't have any side effects, of course.

Best, and hope it helps!
Lucas

### CODE REVIEW

**Review by sophiamaedler — APPROVED** [review]
Thanks for the PR. Looks good :)
